inc/api.php

```
@@ -337,7 +337,57 @@ function djz_get_gamipress_user_data($request) {
+            $event_slugs = apply_filters('djz_dashboard_event_slugs', $default_slugs);
+
+            // Fetch all relevant orders for the user
+            $orders = wc_get_orders([
+                'customer_id' => $user_id,
+                'limit' => -1,
+                'status' => ['completed', 'processing'],
+                'return' => 'ids',
+            ]);
+
+            if (!empty($orders)) {
+                foreach ($orders as $order_id) {
```

PHPMD (2.15.0)

[error] The variable $order_id is not named in camelCase. (undefined)

(CamelCaseVariableName)